### ISSUE TITLE
Create conda package

### ISSUE THREAD

**Issue by huard** [CSHS-CWRA/RavenPy#42]
### Description

At the moment, the conda environment.yml does not install RavenPy binaries, which complicates installation for new users. Also, we'll need an installation mechanism for windows. I suggest we create a conda package that will solve this.

**Comment by tlvu** [comment]
+1 on this as well.  I had to do some contortions to install RavenPy with binaries into the Jupyter env using pip (PR coming soon and you'll see).

**Comment by Zeitsperre** [comment]
@tlvu I can lend a hand on this once you present how you're installing the binaries with pip.

**Comment by cjauvin** [comment]
The Raven binaries are downloaded and compiled using this customized `install` setuptools class: https://github.com/CSHS-CWRA/RavenPy/blob/master/setup.py#L45

This has always been very experimental, and of course I'm very open to suggestions.

**Comment by tlvu** [comment]
> @tlvu I can lend a hand on this once you present how you're installing the binaries with pip.

@Zeitsperre 

Here's my branch for the Jupyter env rebuild: https://github.com/Ouranosinc/PAVICS-e2e-workflow-tests/compare/new-docker-build-with-ravenpy-for-raven-notebooks.

Note it's working now so I am not blocked on this.  If you have other priorities, this can wait but it's certainly a nice to have.

**Comment by cjauvin** [comment]
Note that doing it with pip is not strictly necessary. It's also possible to do it separately, and have the binaries on the path (or supply values for `RAVENPY_{RAVEN|OSTRICH}_BINARY_PATH`) at runtime.

**Comment by tlvu** [comment]
Note that I discovered after the fact that the old Raven also had to do the same contortions https://github.com/Ouranosinc/raven/blob/aeb4ada2d9c9ce27214cdf8662669554df1017af/Dockerfile#L7-L8 so it's not a new problem.

**Comment by cjauvin** [comment]
yeah this is an OS-level dep (like the ones that make us suffer with GDAL) so I don't think we will find a magical solution for it, conda or otherwise: it will need to be dealt in probably vastly different ways on Win, Mac, Solaris, HP-UX, etc.

**Comment by Zeitsperre** [comment]
Closed with https://github.com/conda-forge/ravenpy-feedstock
